ai2_internal/timo/vila/test_fixtures/test_blocks.json

```diff
@@ -0,0 +1 @@
+{"groups": [{"boxes": [{"left": 0.22829851138046364, "top": 0.39771683047516176, "width": 0.5444175371157578, "height": 0.21800039272115687, "page": 0}], "type": "Text"}, {"boxes": [{"left": 0.17708243575750612, "top": 0.6380726592709319, "width": 0.645345301409952, "height": 0.11416552765200837, "page": 0}], "type": "Text"}, {"boxes": [{"left": 0.1833843779719733, "top": 0.7508946428395281, "width": 0.6508047066482843, "height": 0.07361510305693655, "page": 0}], "type": "Text"}, {"boxes": [{"left": 0.18469316819134882, "top": 0.8286216042258523, "width": 0.6653114268982333, "height": 0.07351931177004419, "page": 0}], "type": "Text"}, {"boxes": [{"left": 0.46725164675245096, "top": 0.3638638583096591, "width": 0.06918848573771957, "height": 0.014146631414240057, "page": 0}], "type": "Title"}, {"boxes": [{"left": 0.18218301012625102, "top": 0.11883464967361604, "width": 0.644795985003702, "height": 0.0535144516915986, "page": 0}], "type": "Title"}, {"boxes": [{"left": 0.21118448294845282, "top": 0.24982265510944404, "width": 0.5897238737617443, "height": 0.08330792128437697, "page": 0}], "type": "Text"}]}
```

Comment:
minor, but just for ease of viewing, can u save this JSON w/ `json.dump(..., indent=4)` or something like that?

Reply:
👍